src/M5Stack.h

```diff
@@ -133,6 +133,7 @@ class M5Stack {
 
     // MPU9250
 #ifdef MPU9250_INSDE
+    #include "utility/MPU9250.h"
     MPU9250 IMU = MPU9250();
```

Comment:
I'm not sure if this is disabled deliberately, since [this line](https://github.com/m5stack/M5Stack/pull/120/files#diff-a4632cf0e1ca49c11db93383f7b79235L88) is commented out